include/exec/cpu-all.h

```diff
@@ -261,6 +261,10 @@ extern intptr_t qemu_host_page_mask;
 /* FIXME: Code that sets/uses this is broken and needs to go away.  */
 #define PAGE_RESERVED  0x0020
 #endif
+#ifdef TARGET_CHERI
+#define PAGE_LC_CLEAR	0x8000
+#define PAGE_SC_TRAP	0x2000
```

Comment:
Maybe add a comment for these flags? It took me a few seconds to figure out what LC_CLEAR means. Maybe `PAGE_LC_CLEAR_TAG`?